src/hotspot/share/classfile/symbolTable.cpp

```diff
@@ -237,7 +237,7 @@ void SymbolTable::item_removed() {
 }
 
 double SymbolTable::get_load_factor() {
-  return (double)_items_count/_current_size;
+  return (double)_items_count/(double)_current_size;
```

Comment:
Is this redundant? Both operands are `size_t` so you can cast the result as opposed to both operands.

Reply:
If I cast the result, I'll lose the remainder.  I had to cast current_size because otherwise the compiler complained about it.

Reply:
Yep an unsigned 64-bit type can't be implicitly converted to double without risking a loss of value